src/ffi/mod.rs

```diff
@@ -499,6 +517,8 @@ pub extern "C" fn telio_set_private_key(dev: &telio, private_key: *const c_char)
 
 #[no_mangle]
 pub extern "C" fn telio_get_private_key(dev: &telio) -> *mut c_char {
+    telio_log_info!("telio_get_private_key entry with instance id: {}", dev.id);
```

Comment:
There is not __func__ or something like that in rust?

Reply:
there are smarter methods to log this (and in fact it is done with UniFFI), but on v4.3.3 "manual" method has been used, and I do not feel like now is the time to migrate to another method of logging FFI in v4.3 line.